### DISCUSSION TITLE
Alt Drag, move windows

### DISCUSSION THREAD

**Discussion by 0x70707070** [rxhanson/Rectangle#1107] [Ideas]
Hi, btw great app nice work!!I I think an alt drag function would be pretty nice. Like, keyboard shortkey + mouse to to move the window app.

Example:
![image](https://user-images.githubusercontent.com/32910967/224509701-254b47d1-266f-4d1a-8b00-03622169daad.png)

Thanks.

**Comment by rxhanson** [comment]
Thanks for the feedback. I don't have plans to add this into Rectangle, which is purposefully slim. 

It does exist in [Rectangle Pro](https://rectangleapp.com/pro) (paid). It's in the Window Throw tab (the first tab) of the prefs window toward the bottom.
